app/console.go

```diff
@@ -83,6 +92,92 @@ func RunConsole(a *App) *ishell.Shell {
 	return shell
 }
 
+func encrypt(ctx *ishell.Context, app *App, password string) error {
```

Comment:
Might want to call this function `encryptUser` or something less generic-sounding, or add a comment saying what it does.

Reply:
Same with `decrypt`.